internal/common/gamestate/gamestate_test.go

```diff
@@ -60,6 +60,7 @@ func TestGetClientGameStateCopy(t *testing.T) {
 				ClientLifeStatuses: lifeStatuses,
 				CommonPool:         gameState.CommonPool,
 				IslandLocations:    gameState.Environment.Geography.Islands,
+				IIGORolesBudget:    map[shared.Role]shared.Resources{},
```

Comment:
Rather than doing a direct copy paste. You'd want to do 
```suggestion
				IIGORolesBudget:    gamestate.IIGORolesBudget,
```